bitnami/discourse/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: discourse
+description: A Helm chart for deploying Discourse to Kubernetes
+type: application
+version: 0.1.0
+appVersion: 2.4.1
+home: https://www.discourse.org/
+sources:
+- https://github.com/spinnaker
```

Comment:
Could you also indent these arrays? Our automated lint system would fail otherwise.